### ISSUE TITLE
No rendered video in inference_result folder (Linux)

### ISSUE THREAD

**Issue by aroncserveni** [Inferencer/LipSick#31]
Every frame is synthesized, process is complete but there's no rendered video in the ./asserts/inference_result folder.

Do you have an idea why it doesn't produce the .mp4 result?

<img width="405" alt="Screenshot 2024-06-28 at 22 02 14" src="https://github.com/Inferencer/LipSick/assets/150075609/e5b5f62e-efe5-4b8a-8646-974a5ae7bde2">

**Comment by Inferencer** [comment]
Possible issue with ffmpeg ensure its detected in your env 
I hid the ffmpeg print results so that's why your not seeing an error
while the env is activated you can check if ffmpeg is correctly detected with the following command
`ffmpeg -version`

**Comment by aroncserveni** [comment]
Thank you, ffmpeg was indeed not correctly installed/detected. Now it works.

I have another question. The downloaded result from ./asserts/inference_result is of a much lower quality than the original video I uploaded. Is there a possibility to increase the output quality?

**Comment by Inferencer** [comment]
Yeah comes down to either how torch is installed or which version I think, I was messing around with it a couple of weeks ago and got the audio model to use GPU but it decreased the speed of the frame inference by a lot, resolution quality did return, I will do a pip list tomorrow when I'm on my PC and see what I did.

**Comment by aroncserveni** [comment]
Thank you, I appreciate it

**Comment by aroncserveni** [comment]
> Yeah comes down to either how torch is installed or which version I think, I was messing around with it a couple of weeks ago and got the audio model to use GPU but it decreased the speed of the frame inference by a lot, resolution quality did return, I will do a pip list tomorrow when I'm on my PC and see what I did.

Have you had a chance to make a pip list?

**Comment by Inferencer** [comment]
appears to be torch 1.12
I must have changed my cuda version but wont be able to tell which version due to more changes since then, whatever works with torch 11.2 i suppose https://pytorch.org/get-started/previous-versions/

**Comment by aroncserveni** [comment]
> appears to be torch 1.12 I must have changed my cuda version but wont be able to tell which version due to more changes since then, whatever works with torch 11.2 i suppose https://pytorch.org/get-started/previous-versions/

Thank you! The following seems to have fixed it.

CUDA 11.6
torch==1.12.1
torchvision==0.13.1
torchaudio==0.12.1

When could we expect V2 to be released?
